backend/backend.go

```diff
@@ -517,19 +517,19 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 		coin = btc.NewCoin(coinpkg.CodeLTC, "Litecoin", "LTC", coinpkg.BtcUnitDefault, &ltc.MainNetParams, dbFolder, servers,
 			"https://blockchair.com/litecoin/transaction/", backend.socksProxy)
 	case code == coinpkg.CodeETH:
-		etherScan := etherscan.NewEtherScan("https://api.etherscan.io/api", backend.etherScanHTTPClient)
+		etherScan := etherscan.NewEtherScan("https://api.etherscan.io/v2/api?chainid=1", backend.etherScanHTTPClient)
```

Comment:
I think it's more proper to encode the chainID as a param in the `call()` function and not pass it as part of the url in here. In fact, you can drop the base url param as it's now the same for all. Rather, pass the chainID as an argument.

Reply:
ahahah, I knew you'd want to do it that way :D

Reply:
:joy: It's not the first time you say this either :joy:

Reply:
I changed it a few days ago btw